### PR TITLE
Added ConFoo 2024 cfp

### PR DESCRIPTION
**Pull request by ylarrivee**

### CODE REVIEW

**Comment by ylarrivee** [comment]
Thanks for revewing :)

File: archive/entries/2023-08-20-1.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <content type="xhtml">
+	  <div xmlns="http://www.w3.org/1999/xhtml">
+		  <p>
+			  <a href="https://confoo.ca/en/2024">ConFoo</a> is a multi-technology conference specifically crafted for developers. With over 150 presentations, offered by local and international speakers, the conference wish to bring outstanding diversity of content to expand your knowledge, increase developer’s skills and productivity.
+		  </p>
+		  <p>
+			  Want to share your knowledge and expertise to hundreds of experts in your field? Want to be part of our talented team of renowned speakers? The team at ConFoo is looking for speakers and conference proposals for its 2024 event in Montreal.
+		  </p>
+		  <p>
+			  Our talks are both dynamic and educational and are set to help participants build on their career. We recommend a 45-minute format, including a 10 Q&A session at the end.
```
Comment (adoy) [inline archive/entries/2023-08-20-1.xml:21]:
```suggestion
			  Our talks are both dynamic and educational and are set to help participants build on their career. We recommend a 45-minute format, including a 10 Q&amp;A session at the end. 
```

Comment (adoy) [reply archive/entries/2023-08-20-1.xml:21]: Otherwise lgtm ;)

**Review by ylarrivee — COMMENTED** [review]
Looks good.

**Pull request merged**